firebase.json

```diff
@@ -96,15 +97,21 @@
     "rules": "firestore.rules",
     "indexes": "firestore.indexes.json"
   },
+  "storage": {
+    "rules": "storage.rules"
+  },
   "functions": [
     {
-      "source": "functions",
-      "codebase": "default",
+      "source": "functions/node",
+      "codebase": "node",
+      "ignore": ["node_modules", ".git", "firebase-debug.log", "firebase-debug.*.log", "*.local"],
+      "predeploy": ["pnpm --prefix \"$RESOURCE_DIR\" run build"]
+    },
+    {
+      "source": "functions/python",
+      "codebase": "python",
```

Comment:
it this going to update some metadata or try to delete all the existing functions and move them to python?